_pages/en_US/home.md

```diff
@@ -21,38 +19,42 @@ Thoroughly read all of the introductory pages (including this one!) before proce
 ## What is homebrew?
 
-[Homebrew](https://en.wikipedia.org/wiki/Homebrew_(video_games)) on the DSi usually refers to software that is not authorized by Nintendo. It allows you to run homebrew games, tools like save editing and backup, and emulators for various older systems.
+[Homebrew](https://en.wikipedia.org/wiki/Homebrew_(video_games)) applications are unlicensed software made for closed systems such as the Nintendo DSi. These applications can range from utilities to custom homebrew games.
 
-Homebrew can be run for free on all Nintendo DSi consoles, regardless of firmware version or region. All you need is an entry point and an SD card to store your homebrew. The main entry point used in this guide will be called Memory Pit, but there are other entry points you could use if Memory Pit is unusable.
+Homebrew can be run for free on all Nintendo DSi consoles, regardless of firmware version or region. All you need is an entry point and an SD card to store your homebrew. The main entry point used in this guide is called Memory Pit, but there are other entry points you can use if Memory Pit is unusable.
 
-## What is Custom Firmware?
+## What will this guide do to my system?
 
-**Custom Firmware** ("CFW") is a modified version of the original firmware that extends functionality and makes improvements to certain aspects. For instance, it can be used to creating backups of games and run previously blocked flashcarts. You can also configure it to run homebrew titles at boot by holding down a specific button.
+Keep in mind that the guide is structured in a linear path, yet it generally depends on how deep you want to go in terms of modding the system.
 
-Custom Firmware on a Nintendo DSi can be launched using a low-level exploit, called Unlaunch. From there, you can install hiyaCFW, an aforementioned CFW which runs your internal storage (NAND) off of the SD card, dubbed "SDNAND." This allows for the installation of custom homebrew and DSiWare apps onto your DSi Menu without the risks of breaking your system.
-
-The benefits of Unlaunch + hiyaCFW over other homebrew launch methods are numerous and as such, it is recommended to use this guide over any other that relies on outdated software (such as sudokuhax, hbmenu and hardmodding).
-
-The only requirement to installing Custom Firmware is to make sure you can access homebrew, which we will get to in the guide.
+- We will be setting up an entry point to launch basic homebrew via Memory Pit, the exploit for the Nintendo DSi Camera application
+- We will then take a NAND backup, which is useful as a restoration point in case you mess up later on
+- For users that want more control, we will walk them through installing Unlaunch. Unlaunch allows homebrew to run with more permissions (such as slot-1 access) and at system boot
+   - Installing Unlaunch modifies your NAND, and in extreme edge cases has the potential to **brick** the system. If you do not risk bricking, you can stop after taking a NAND backup
```

Comment:
```suggestion
   - Installing Unlaunch modifies your NAND and in rare circumstances can **brick** the system. If you do not want to take that risk, you can stop after making a NAND backup
```
Were talking about this on Discord, I'm gonna say this as my favorite so far but comment other ideas too if you want